specification/digitaltwins/data-plane/Microsoft.DigitalTwins/stable/2020-10-31/digitaltwins.json

```diff
@@ -0,0 +1,1583 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-10-31",
+    "title": "Azure Digital Twins API",
+    "description": "A service for managing and querying digital twins and digital twin models."
+  },
+  "host": "digitaltwins-name.digitaltwins.azure.net",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "paths": {
+    "/models": {
+      "post": {
+        "tags": [
+          "Models"
+        ],
+        "description": "Uploads one or more models. When any error occurs, no models are uploaded.\nStatus codes:\n200 (OK): Success.\n400 (Bad Request): The request is invalid.\n409 (Conflict): One or more of the provided models already exist.",
+        "operationId": "DigitalTwinModels_Add",
+        "x-ms-examples": {
+          "Add models": {
+            "$ref": "./examples/ModelAdd.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "models",
+            "in": "body",
+            "description": "An array of models to add.",
+            "required": false,
+            "schema": {
+              "type": "array",
+              "minItems": 1,
+              "uniqueItems": true,
+              "items": {
+                "type": "object"
+              }
+            }
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/traceparent"
+          }
+        ],
+        "responses": {
+          "201": {
```

Comment:
Of course, using POST to create a resource is not idempotent and if a client retries this, then the 201 could be ignored entirely which is another reason why 201 is problematic in general.